packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md

```diff
@@ -37,47 +37,47 @@ Scoped access tokens have several advantages, including:
 
 ## Create an OAuth 2.0 app in Okta
 
-Create the client application that you want to use with the Okta APIs.
-
-1. Sign in to your Okta organization as a user with administrative privileges. [Create an org for free](https://developer.okta.com/signup).
+Create the client app that you want to use with the Okta APIs.
 
+1. Sign in to your Okta organization with your administrator account.
 1. In the Admin Console, go to **Applications** > **Applications**.
 1. Click **Create App Integration**.
-1. On the **Create a new app integration** page, select **OIDC - OpenID Connect** as the **Sign-in method**. Choose **Web Application** for the **Application type**. Creating a web app is an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token. Click **Next**.
+1. On the **Create a new app integration** page, select **OIDC - OpenID Connect** as the **Sign-in method**.
+1. Choose **Web Application** for the **Application type**. Creating a web app is an easy way to test scope-based access to the Okta APIs using an OAuth 2.0 bearer token. Click **Next**.
 
-    > **Note:** It's important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients aren't designed to have, and would break the sign-in or sign-out flow.
+    > **Note:** It's important to choose the appropriate app type for apps that are public clients. Not choosing the right app may result in Okta API endpoints attempting to verify an app's client secret. Public clients aren't designed to have these, and so it breaks the sign-in or sign-out flow.
```

Comment:
```suggestion
    > **Note:** It's important to choose the appropriate app type for apps that are public clients. Choosing the wrong app type may result in Okta API endpoints attempting to verify an app's client secret. Because public clients aren't designed to have these, the sign-in or sign-out flow can break.
```